src/MSBuild/Resources/Strings.resx

```diff
@@ -722,6 +722,22 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
     </comment>
   </data>
+  <data name="HelpMessage_35_GraphBuildSwitch" UESanitized="false" Visibility="Public">
+    <value>/graphBuild
```

Comment:
Tests are failing because this didn't copy the two space characters from the other messages. Nice that someone actually added tests for indentation consistency . . .

Reply:
Yea, I should run tests before embarrassingly sending out PRs with failing tests :)